cypress/support/fill.ts

```diff
@@ -10,16 +9,15 @@ declare namespace Cypress {
   interface Chainable {
     /**
      * Custom command to check an external Link.
```

Comment:
```suggestion
     * Custom command to fill an input field with text and trigger a change event.
```

Sorry to bother you again. I overlooked this at first.

Reply:
done